### PR TITLE
generating 551 release notes, and editing similar issues in the 5.6 r…

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
…elease notes.

### CODE REVIEW

File: docs/release_notes_128t_5.5.md [hunk 1]
```diff
@@ -24,6 +24,68 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-45113 snmp override of the IfTable:** An issue with SNMP reporting has been resolved.
+------
+- **I95-45124 RBAC Config Endpoints Leaking Information:** Resolved an issue where some configuration endpoints would allow users with incorrect permissions make requests.
+------
+- **I95-45171 Mist Service Route is not auto-generated when Mist WAN-Assurance is enabled by default:** This issue has been resolved.
+------
+- **I95-45348 Update salt master and minion to 3002.8:** This update resolves several CVE's and requires that the conductor must be running this release containing these fixes **before** upgrading a router. 
+**Important** Please see the Caveat below for additional important information about HA upgrades.
+------
+- **I95-45374 Router Dropping SIP traffic:** A warning is displayed if users configure a service-class to rate-limit but don't set max-flow-burst/max-flow-rate values (default is set to 0).
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:67]: Heading should read: "Service-class with a rate-limit of 0 will drop traffic"

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_5.5.md [hunk 2]
```diff
@@ -24,6 +24,68 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-45514 Only allow DSCP-steering config on forwarding interfaces:** DSCP Steering config no longer appears on non-forwarding interfaces.
+------
+- **I95-45541 LDAP users are unable to login to the PCLI due to permission errors:** This issue has been resolved.
+------
+- **I95-45696 Memory leak in pam challenge library:** Resolved a memory leak in the PAM challenge library. 
+------
+- **I95-45767 Verify RBAC handles spaces and escaped characters correctly:** Both spaces and escaped characters are properly handled.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md:75]: I think this is one that probably does not need to go into the release notes. The Jira issue was to verify that it indeed did handle them correctly, not that it didn't and there was a problem. I'm going to remove it from the R/Ns.

File: docs/release_notes_128t_5.5.md [hunk 3]
```diff
@@ -24,6 +24,68 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-45842 PCLI `show events` does not paginate correctly:** This issue has been resolved.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md:76]: done

File: docs/release_notes_128t_5.5.md [hunk 4]
```diff
@@ -24,6 +24,68 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-45882 Rare case where an invalid DHCP server configuration generated:** This issue has been resolved.
+------
+- **I95-46055 Add warning when transmit caps are too low:** Users now get a warning when configuring a traffic-engineering transmit-cap under 1Mbps.
+------
+
+### Caveats
+
+- **I95-45348: Update salt master and minion to 3002.8:** When upgrading an HA pair to version 5.5.1, please be aware of the following: While updating the conductors in an HA pair, the upgraded conductor node asset state will remain DISCONNECTED if the active `automatedProvisioner` is not running a corrected version. When performing an HA conductor upgrade the node running the oldest software assumes leadership. However, the older version will not be able to talk to the new software on the upgraded conductor.
```
[the remark this answers is not in the document]
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.5.md:85]: Added the appropriate info

**Review by MichaelBaj — DISMISSED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

File: docs/release_notes_128t_5.5.md [hunk 5]
```diff
@@ -41,16 +41,48 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
 ------
 - **I95-43779 DHCP IP Address is not released:** Updated the state machine to cause DHCP-enabled interfaces to send out a DHCP Request for their current IP address.
 ------
+- **I95-43897 Planned failover did not work properly:** Resolved an issue where a waypoint missing from an internal database prevented failover.  
+------
 - **I95-44001 Peer uptime showing "Unavailable":** Peer path uptime now displays the correct values.
 ------
+- **I95-44142 Automated Provisioner race condition:** Resolved a rare crash where applications would attempt to get information about already-closed sockets when responding to API requests.
+------
+- **I95-44443 NTP Server config not always picked up:** Resolved an issue where NTP configuration was changed but the backend would not take action on those changes.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_5.5.md:74]: This could be made more succinct by stating ntp server configuration is not dynamically reconfigurable.

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**